library/core/src/mem/maybe_uninit.rs

```diff
@@ -999,7 +1003,11 @@ impl<T> MaybeUninit<T> {
     ///
     /// [`assume_init_mut`]: MaybeUninit::assume_init_mut
     #[unstable(feature = "maybe_uninit_slice", issue = "63569")]
-    #[rustc_const_unstable(feature = "const_maybe_uninit_assume_init", issue = "none")]
+    #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
+    #[rustc_const_stable(
+        feature = "const_maybe_uninit_assume_init",
+        since = "CURRENT_RUSTC_VERSION"
+    )]
```

Comment:
This is still unstable, let's just use the same feature gate for both.
```suggestion
    #[rustc_const_unstable(
        feature = "maybe_uninit_slice",
        issue = "63569"
    )]
```

Reply:
Ah, sure.